rpc/src/v1/impls/chain.rs

```diff
@@ -237,7 +238,7 @@ where
     }
 
     fn get_min_transaction_fee(&self, action_type: String, block_number: u64) -> Result<Option<u64>> {
-        if let Some(common_parameters) = self.client.common_params(Some(block_number)) {
+        if let Some(common_parameters) = self.client.common_params(block_number.into()) {
```

Comment:
Should we use parent's block common params?
If the RPC's caller wants to fee to creates transactions in the next block, the current implementation is OK.
However, if the RPC's caller wants to check the block's min_fee to verify the block, we should use the previous block's common params.

Reply:
I make it use the parameters of the parent block.